op-proposer/flags/flags.go

```diff
@@ -54,7 +54,6 @@ var (
 		Name:    "dgf-address",
 		Usage:   "Address of the DisputeGameFactory contract",
 		EnvVars: prefixEnvVars("DGF_ADDRESS"),
-		Hidden:  true,
```

Comment:
I like the idea of making flags available in the help menu, so we should remove the `Hidden` field for the `ProposalIntervalFlag` and `DisputeGameTypeFlag` as well.

IMO the best way to do this is to update the `Usage` fields for all of these, appending `[Only When Fault Proofs Enabled].` or something to that effect.

Reply:
We can also introduce flag sections/categories. I've been wanting to do that for op-node. We can do this by setting the Category field on these flags.

Reply:
Yeah I'd meant to unhide them all, not sure how I missed those.

Categories sounds like a very useful idea but I'm not going to attempt that this stage to stay focussed. It won't be that long and we'll remove the old L2OO options anyway.

I've also tweaked the names a bit to avoid the abbreviations and make the easier to understand.